### ISSUE TITLE
Add reference link to PgnJS?

### ISSUE THREAD

**Issue by xeyownt** [mliebelt/PgnViewerJS#70]
I started developing a mediawiki parser extension that gives access to your JS engine through the tag `<pgn>`. Have a look at [**PgnJS**](https://github.com/xeyownt/mwpgnjs).

For now, the implementation is only a proof-of-concept. I plan to extend it in the coming days.

If you have any comment or suggestion, please don't hesitate.

**Comment by xeyownt** [comment]
For your information, I just switched the extension from *experimental* status to *beta* status. I think it contains enough stable features to do the switch. To summarize:
* (stable) all publicly documented PgnViewerJS features.
* (experimental) attribute `goto`.
* (broken) attributes `movesWidth`, `movesHeight` and `scrollable`.

I also created a test server to demonstrate the extension and test all features. People can visit this server to try the extension by themselves. Have a look at [the PgnJS page on WikiTest](https://wikitest.immie.org/wiki/PgnJS).

**Comment by mliebelt** [comment]
I have read your page, and it looks impressing! So glad that the PgnViewerJS is helpful for you. I just released version 0.9.6, which should fix the movesWidth, movesHeight, ... problem by using the correct parameter layout. There are multiple examples available, how to use the layout with the correct sizes. 

for the goto, we should find a simple, understandable solution (like using the move number from the notation). This will follow soon, I think.

**Comment by xeyownt** [comment]
Indeed, I was missing the attribute `layout` ! I just made a new release with this attribute added and version 0.9.6, and this fixed all broken features indeed. I find the `layout` tag a very nice addition, and I updated my sample wikitest page to demonstrate it better.

Also if I may, please don't forget the pull request. Currently I point to my fork of PgnViewerJS because I need a small patch on the script path retrieval. If you want to update the changes, it would be nice to still merge the proposed commits so that not to break the git submodule reference in PgnJS.

**Comment by xeyownt** [comment]
For your information, I just issued yet another release (I'll stop soon...), mainly to deal with MediaWiki related features, namely localization and managing defaults:
* Extension now reads the user preferences to get the defaults for piece style, theme, timer time and locale.
* I defined a fifth (experimental feature) mode, `defaults`, that can be used to define new defaults, either for all boards, or for boards with a specific classname, as given by attribute `class`. Several classnames can be given for full customization.
I updated the wikitest page to illustrate all this.
